src/fs_event_wrap.cc

```diff
@@ -86,16 +88,20 @@ void FSEventWrap::Start(const FunctionCallbackInfo<Value>& args) {
 
   FSEventWrap* wrap = Unwrap<FSEventWrap>(args.Holder());
 
-  if (args.Length() < 1 || !args[0]->IsString()) {
-    return env->ThrowTypeError("filename must be a valid string");
-  }
+  const char* kErrMsg = "filename must be a string or Buffer";
```

Comment:
Can you write this as `static const char kErrMsg[] = ...`?  Avoids the extra pointer indirection.  (Although it will probably get optimized away anyway but it's more obviously "right.")